pytorch_lightning/utilities/parsing.py

```diff
@@ -161,6 +162,47 @@ def flatten_dict(source, result=None):
     return result
 
 
+def save_hyperparameters(
+    obj: Any,
+    *args,
+    ignore: Optional[Union[Sequence[str], str]] = None,
+    frame: Optional[types.FrameType] = None
+) -> None:
+    """See :meth:`~pytorch_lightning.core.lightning.LightningModule.save_hyperparameters`"""
```

Comment:
Please change the reference.

Reply:
It is fine. If readers get there, they can re-direct themselves to the LightningModule.